flocker/provision/_install.py

```diff
@@ -114,6 +114,51 @@ def task_upgrade_kernel():
     ]
 
 
+KOJI_URL_TEMPLATE = (
+    'https://kojipkgs.fedoraproject.org/packages/kernel'
+    '/{version}/{release}.{distribution}/{architecture}'
+    '/kernel-{version}-{release}.{distribution}.{architecture}.rpm'
+)
+
+
+def koji_kernel_url(kernel):
+    """
+    Return the koji URL for the given kernel version.
+    """
+    url = KOJI_URL_TEMPLATE.format(
+        version=kernel.version,
+        release=kernel.release,
+        distribution=kernel.distribution,
+        architecture=kernel.architecture
+    )
+    return url
+
+
+DIGITALOCEAN_KERNEL = Kernel(
+    version="3.17.8",
+    release="200",
+    distribution="fc20",
+    architecture="x86_64"
+)
+
+
+DIGITALOCEAN_KERNEL_TITLE = (
+    "Fedora 20 x64 "
+    "vmlinuz-{kernel.version}-{kernel.release}"
+    ".{kernel.distribution}.{kernel.architecture}"
+).format(kernel=DIGITALOCEAN_KERNEL)
+
+
+def task_install_digitalocean_kernel():
```

Comment:
:question: It would probably make more sense for this to just take a `Kernel` to install. I guess that makes the sphinx more complicated, though.

I was going to suggest that the above to `DIGITIALOCEAN_` constants be moved to `._digitalocean`.